starter-code/public/scripts/article.js

```diff
@@ -60,13 +65,13 @@ Article.fetchAll = callback => {
 
 // TODO: Chain together a `map` and a `reduce` call to get a rough count of all words in all articles.
 Article.numWordsAll = () => {
-  return Article.all.map().reduce()
+  return Article.all.map(article => {return article.body.split(' ').length}).reduce((total, current) => {return total + current;}, 0)
```

Comment:
So when trying to do one line arrow functions, you dont need a return or the '{}', it would look like (something => something), or in this case .map(article => article.body.split(' ').length)
Same goes for the reduce